asciidoctorj-core/src/test/java/org/asciidoctor/WhenAsciiDocIsRenderedToDocument.java

```diff
@@ -146,7 +149,7 @@ public void should_be_able_to_get_roles() {
         AbstractBlock abstractBlock = document.blocks().get(0);
         assertThat(abstractBlock.getRole(), is("famous"));
         assertThat(abstractBlock.hasRole("famous"), is(true));
-        //assertThat(abstractBlock.isRole(), is(true));
```

Comment:
I think it was because of bug in Asciidoctor which didn't return correctly. Well if I remember well it was a problem on how the method was named in Ruby side that didn't match so well in Java part.